extensions/ql-vscode/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [UNRELEASED]
 
 - Re-enable publishing to open-vsx. [#1285](https://github.com/github/vscode-codeql/pull/1285)
+- Databases downloaded from GitHub have a more user friendly name. These databases will now be named `{org}/{repo}` instead of `codeql-db`. [#1294](https://github.com/github/vscode-codeql/pull/1294)
```

Comment:
No need for a changelog entry - downloading from GitHub is still a hidden feature (due to the MRVA API being restricted)

Reply:
Right!